tests/scripts/task_config_build_cpu.sh

```diff
@@ -45,3 +45,4 @@ echo set\(USE_TENSORFLOW_PATH \"/tensorflow\"\) >> config.cmake
 echo set\(USE_FLATBUFFERS_PATH \"/flatbuffers\"\) >> config.cmake
 echo set\(USE_ETHOSN /opt/arm/ethosn-driver\) >> config.cmake
 echo set\(USE_ETHOSN_HW OFF\) >> config.cmake
+echo set\(USE_OPENCL ON\) >> config.cmake
```

Comment:
This will need to go as a separate PR, and be merged before you submit this one.

Please create a new PR and cc @tqchen

Reply:
Created a PR https://github.com/apache/incubator-tvm/pull/6847/files